src/memray/reporters/flamegraph.py

```diff
@@ -7,10 +7,13 @@
 from typing import Iterable
 from typing import List
 from typing import Optional
+from typing import Sequence
 from typing import TextIO
 from typing import Tuple
+from typing import TypedDict
```

Comment:
CI is failing because of this. We still support 3.7, and `typing.TypedDict` wasn't added until Python 3.8. We either need to use `from typing_extensions import TypedDict` on Python 3.7, or drop support for Python 3.7 - or maybe make this conditional on Python version... or maybe only use `TypedDict` when `typing.TYPE_CHECKING` is set.